src/hooks/useCookieConsent.ts

```diff
@@ -0,0 +1,17 @@
+import { useEffect } from 'react'
+
+export const useCookieConsent = () => {
+  useEffect(() => {
+    const script = document.createElement('script')
+
+    script.src = 'https://app.usercentrics.eu/browser-ui/latest/bundle.js'
+    script.defer = true
+    script.id = 'usercentrics-cmp'
+    script.setAttribute('data-settings-id', 'MCpNm4tDT')
```

Comment:
Should this be an env var?

Reply:
Contentful-Gatsby key -> GATSBY_USERCENTRICS_ID